core/common/src/main/java/com/google/samples/apps/nowinandroid/core/result/Result.kt

```diff
@@ -27,11 +27,20 @@ sealed interface Result<out T> {
     object Loading : Result<Nothing>
 }
 
-fun <T> Flow<T>.asResult(): Flow<Result<T>> {
+fun <T, R> Flow<T>.mapToResultUiState(
```

Comment:
Not really a fan of this as UI state isn't something which the Result class should care about.